index.html

```diff
@@ -2181,6 +2181,159 @@ <h4>Presentations Including Holder Claims</h4>
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://university.example/credentials/3732",
+  "type": ["VerifiableCredential", "ExampleDegreeCredential", "ExamplePersonCredential"],
+  "issuer": "https://university.example/issuers/14",
+  "validFrom": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "ExampleBachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    },
+    "alumniOf": {
+      "name": "Example University"
+    }
+  },
+  <span class="highlight">"credentialSchema": [{
+    "id": "https://example.org/examples/degree.json",
+    "type": "JsonSchema2023"
+  },
+  {
+    "id": "https://example.org/examples/alumni.json",
+    "type": "JsonSchema2023"
+  }]</span>
+}
+        </pre>
+
+        <p>
+In the example above, the <a>issuer</a> is specifying a
+<code>credentialSchema</code>, which points to a [[?VC-JSON-SCHEMA-2023]] file that
+can be used by a <a>verifier</a> to determine if the
+<a>verifiable credential</a> is well formed.
+        </p>
+
+        <p class="note" >
+For information about linkages to JSON Schema [[?VC-JSON-SCHEMA-2023]] or other
+optional <a>verification</a> mechanisms, see the Verifiable Credentials
+Implementation Guidelines [[VC-IMP-GUIDE]] document.
+        </p>
+
+        <p>
+Data schemas can also be used to specify mappings to other formats, such as
+those used to perform zero-knowledge proofs. For more information on using the
+<code>credentialSchema</code> <a>property</a> with zero-knowledge proofs,
+see Section <a href="#zero-knowledge-proofs"></a>.
+        </p>
+
+        <pre class="example nohighlight" title="Usage of the credentialSchema property to perform zero-knowledge validation">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://university.example/credentials/3732",
+  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
+  "issuer": "https://university.example/issuers/14",
+  "validFrom": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "ExampleBachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    }
+  },
+  <span class="highlight">"credentialSchema": {
+    "id": "https://example.org/examples/degree",
+    "type": "ZkpExampleSchema2018"
+  }</span>,
+  "proof": { <span class="comment">...</span> }
+}
+        </pre>
+
+        <p>
+In the example above, the <a>issuer</a> is specifying a
+<code>credentialSchema</code> pointing to a means of transforming the input
+data into a format which can then be used by a <a>verifier</a> to determine if
+the proof provided with the <a>verifiable credential</a> is valid.
+        </p>
+
+
```

Comment:
```suggestion
```

Reply:
Why are we speculating like this in the core data model?

We have no work items that "perform this mapping"... this entire section is hypothetical.

Either make this concrete (with citations), or remove it.

Reply:
agree this should be removed.